### PR TITLE
291 feature request transaction filters

### LINKED ISSUE #291: Feature Request - Transaction Filters

### PR DESCRIPTION
**Pull request by WikusKriek**
# Pull Request Template

## Description

Transaction table filters. 
Added continious requests to get latest transaction updates on network page.
fixed continious requests does not stop on page navigate.
image
image
image

Fixes #291 

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:
* Firmware version:
* Hardware:
* Toolchain:
* SDK:

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: src/components/TransactionsTable.vue [hunk 1]
```diff
@@ -169,19 +211,63 @@ export default defineComponent({
...
+                template(v-slot:append)
+                  q-icon.cursor-pointer(name='access_time')
+                    q-popup-proxy(cover='' transition-show='scale' transition-hide='scale')
+                      q-time(v-model='toDateFilter' mask='YYYY-MM-DD HH:mm' format24h)
+                        .row.items-center.justify-end
+                          q-btn(v-close-popup='' label='Close' color='primary' flat)
+        q-btn-dropdown.q-ml-xs.q-mr-xs.col.button-primary(
+          color="primary"
+          label="Token")
+          .q-pa-md(style='max-width: 300px')
```
Comment (donnyquixotic) [inline src/components/TransactionsTable.vue:262]: remove inline style to a class

File: src/components/TransactionsTable.vue [hunk 2]
```diff
@@ -169,19 +211,63 @@ export default defineComponent({
...
+          color="primary"
+          label="Actions")
+          .q-pa-md(style='max-width: 300px')
+            .row
+              q-input(filled dense v-model='actionsFilter' label="Actions")
+        q-btn-dropdown.q-ml-xs.q-mr-xs.col.button-primary(
+          persistent
+          color="primary"
+          label="Date")
+          .q-pa-md(style='max-width: 300px')
```
Comment (donnyquixotic) [inline src/components/TransactionsTable.vue:228]: remove inline style to a class

File: src/components/TransactionsTable.vue [hunk 3]
```diff
@@ -169,19 +211,63 @@ export default defineComponent({
...
       q-space
-      div.col-3.row.flex.filter-buttons.temp-hide
-        q-btn.q-ml-xs.q-mr-xs.col.button-primary Actions
-        q-btn.q-ml-xs.q-mr-xs.col.button-primary Date
-        q-btn.q-ml-xs.q-mr-xs.col.button-primary Token
+      div.col-auto.row.flex.filter-buttons
+        q-btn-dropdown.q-ml-xs.q-mr-xs.col.button-primary(
+          color="primary"
+          label="Actions")
+          .q-pa-md(style='max-width: 300px')
```
Comment (donnyquixotic) [inline src/components/TransactionsTable.vue:221]: remove inline style to a class

**Review by donnyquixotic — CHANGES_REQUESTED** [review]

File: src/components/TransactionsTable.vue (hunk 1 above)
Comment (WikusKriek) [reply src/components/TransactionsTable.vue]: Done

**Review by donnyquixotic — APPROVED** [review]

**Pull request merged**